src/styles/_catalog.scss

```diff
@@ -1,22 +1,28 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
-
-  max-width: 944px;
-  margin: 0 auto;
+  display: grid;
+  grid-template-columns: repeat(auto-fill, 200px);
+  justify-content: center;
+  gap: 48px;
 
   &__card {
-    display: flex;
-    justify-content: center;
-    align-items: center;
     width: 200px;
     height: 300px;
+    text-align: center;
+    line-height: 300px;
+    box-sizing: border-box;
     border: 1px solid #ccc;
     border-radius: 5px;
-    margin-bottom: 48px;
   }
+}
+
+@media (max-width: 487.5px) {
```

Comment:
It is better to use one state Mobile First or Desktop First (recommend to use Mobile one) 
[link](https://ishadeed.com/article/the-state-of-mobile-first-and-desktop-first/)
Update your media-queries using `min-width`